infocmdb/ci_type.go

```diff
@@ -93,3 +94,38 @@ func (c *Client) GetCiTypeName(ciId int) (ciTypeName string, err error) {
 
 	return
 }
+
+type respSetTypeOfCi struct {
+	Message bool `json:"success"`
+}
+
+func (c *Client) SetTypeOfCi(ciId int, ciType string) (respStatus string, err error) {
+
+	if err = c.v2.Login(); err != nil {
+		return
+	}
+
+	ciTypeId, err := c.GetCiTypeIdByCiTypeName(ciType)
```

Comment:
It looks like the error here is ignored.
I'd be better to use a early return to prevent calling the "int_setCiTypeOfCi" webservice with wrong arguments.

Reply:
Not ignoring the error anymore.